### PR TITLE
Feature gif disposal option for WMS Animator

### PR DESCRIPTION
**Pull request by MauroIT**
Please see: 

JIRA: https://osgeo-org.atlassian.net/browse/GEOS-7222

Mailing List: http://sourceforge.net/p/geoserver/mailman/geoserver-devel/thread/56040fbe.c755c20a.da03.41dd%40mx.google.com/#msg34487376

Please let me know if you need further information

### CODE REVIEW

File: src/web/wms/src/main/resources/GeoServerApplication_ca.properties [hunk 1]
```diff
@@ -1,103 +1,104 @@
-#Generated by ResourceBundle Editor (http://eclipse-rbe.sourceforge.net)
```
Comment (aaime) [inline src/web/wms/src/main/resources/GeoServerApplication_ca.properties]: The entire file got modified... I believe it's newlines changes. Can you avoid it?

File: src/web/wms/src/main/resources/GeoServerApplication_ja.properties [hunk 2]
```diff
@@ -53,6 +53,7 @@ WMSAdminPage.bboxForEachCRSHelp.title=CRS\u3054\u3068\u306e\u7bc4\u56f2\u77e9\u5
...
 WMSAdminPage.defaultInterpolation   = \u898f\u5b9a\u306e\u88dc\u5b8c
 WMSAdminPage.description            = \u5730\u56f3\u306e\u516c\u958b\u7ba1\u7406
+WMSAdminPage.disposalMethod         = Disposal Method
```
Comment (aaime) [inline src/web/wms/src/main/resources/GeoServerApplication_ja.properties]: Please leave the strings in the other languages as missing, the code should automatically display the english version if the string was not found

File: src/wms/src/test/java/org/geoserver/wms/animate/AnimatorTest.java [hunk 3]
```diff
@@ -132,7 +132,11 @@ public void testAnimator() throws Exception {
         checkAnimatedGif(requestURL, false);
         checkAnimatedGif(requestURL + "&format_options=gif_loop_continuously:true", true);
         checkAnimatedGif(requestURL + "&format_options=gif_loop_continuosly:true", true);
-
+        // check all valid disposal methods
+        checkAnimatedGif(requestURL + "&format_options=gif_disposal:none", false);
```
Comment (aaime) [inline src/wms/src/test/java/org/geoserver/wms/animate/AnimatorTest.java]: The test is not really checking the disposal method got applied, could you add that?

**Comment by aaime** [comment]
Hi Mauro, a few other notices:
* The pull should contain a single commit, with number of the ticket and title
* In order to contribute to GeoServer you or your company need to sign a contribution agreement to GeoServer, see the contribution guide: https://github.com/geoserver/geoserver/blob/master/CONTRIBUTING.md
* I did not check file by file, but make sure that the headers in the files you have modified have the last year in the copyright as 2015, e.g., if you find "(c) 2014 Open Source Geospatial Foundation - all rights reserved" it should become "(c) 2014 - 2015 Open Source Geospatial Foundation - all rights reserved", if it's "(c) 2012 - 2013 Open Source Geospatial Foundation - all rights reserved" it should become "(c) 2012 - 2015 Open Source Geospatial Foundation - all rights reserved"

**Comment by MauroIT** [comment]
Hi Aaime, 
I'm going to work on your named adjustments and create a new pull request.
I've already sent a sign contribution agreement (Maurizio La Mela).

**Comment by aaime** [comment]
Maurizio, I still don't see the agreement... it might be that the OSGeo secretary simply has not filed it yet, but to make sure, which address did you send the agreement to?

**Comment by aaime** [comment]
Never mind, found the agreement on file.

**Comment by MauroIT** [comment]
I've just created a new pull request: https://github.com/geoserver/geoserver/pull/1263

**Pull request closed without merging**